catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrMetacardClientImplTest.java

```diff
@@ -204,14 +214,92 @@ public void testQuerySpellCheckOn() throws Exception {
     verify(queryResponse, times(2)).getResults();
   }
 
+  @Test
+  public void testHighlightOn() throws Exception {
```

Comment:
✏️ I would suggest adding a unit test for XML fields and highlighting since we strip XML tags for the value that is indexed.

Reply:
@vinamartin Did you test highlights on the metadata field by chance during your retest?